recipe/meta.yaml

```
@@ -14,7 +14,7 @@ source:
 
 build:
   number: 1
-  skip: true  # [ppc64le or (cuda_compiler_version != "11.2")]
+  skip: true  # [cuda_compiler_version != ("11.2", "12.0") or (cuda_compiler_version == "11.2" and ppc64le)]
```

Think I had `[]` and not `()` before 🤦‍♂️

```suggestion
  skip: true  # [ppc64le or cuda_compiler_version not in ("11.2", "12.0")]
```